internal/manager/controllerdef.go

```diff
@@ -107,7 +83,7 @@ func setupControllers(logger logr.Logger, mgr manager.Manager, proxy proxy.Proxy
 			},
 		},
 		{
-			IsEnabled:   &c.IngressNetV1beta1Enabled,
+			Enabled:     c.IngressNetV1beta1Enabled,
```

Comment:
nit: does this `gofmt -w` cleanly? I see a whitespace inconsistency here :thinking:

Reply:
Never mind, this is probably OK.